wpilibc/src/main/native/cpp/TimedRobot.cpp

```diff
@@ -110,3 +110,11 @@ void TimedRobot::AddPeriodic(std::function<void()> callback,
       std::chrono::microseconds{static_cast<int64_t>(period.value() * 1e6)},
       std::chrono::microseconds{static_cast<int64_t>(offset.value() * 1e6)});
 }
+
+void TimedRobot::AddPeriodic(std::function<void()> callback,
+                             units::hertz_t period, units::hertz_t offset) {
```

Comment:
This is a frequency, so calling it `period` doesn't make sense. Offset doesn't make sense as a frequency either.